src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -227,10 +227,7 @@ static bool detectSPISensorsAndUpdateDetectionResult(gyroDev_t *gyro, const gyro
 {
     gyro->bus.bustype = BUSTYPE_SPI;
 
-    spiBusSetInstance(&gyro->bus, spiInstanceByDevice(SPI_CFG_TO_DEV(config->spiBus)));
-
-    // SPI instance may be NULL if the bus is non-existent
-    if (!gyro->bus.busdev_u.spi.instance) {
+    if (!spiBusSetInstance(&gyro->bus, spiInstanceByDevice(SPI_CFG_TO_DEV(config->spiBus)))) {
```

Comment:
I'd rather go

```
SPI_TypeDef *instance = spiInstanceByDevice(SPI_CFG_TO_DEV(config->spiBus));
if (!instance) {
￼    return false;
￼}
spiBusSetInstance(&gyro->bus, instance);
```

to make it a bit more obvious what is done. But that's a detail really.
￼
￼

Reply:
Hm. That's a point. @mikeller What do you think?

Reply:
Was that for @DieHertz? ;-)

Reply:
😅